src/formEditor/FormXmlSerializer.ts

```
@@ -0,0 +1,357 @@
+    this.originalXml = xmlContent;
+    this.orderedTree = parser.parse(xmlContent);
+  }
+
+  /** Переместить элемент формы (по id) в новую позицию */
+  moveElement(
+    elementId: number,
+    targetParentId: number,
+    insertBeforeId: number | null
+  ): boolean {
+    const formNode = this.findTag(this.orderedTree, 'Form');
+    if (!formNode) return false;
```

ESLint

[error] Expected { after 'if' condition.

(curly)